### PR TITLE
Feature/ripple billing

### PR DESCRIPTION
**Pull request by stevenzeiler**
Register new Ripple addresses, monitor them for payments, credit contracts upon payment.

### CODE REVIEW

**Comment by wilsonianb** [comment]
Add tracker tag to commit message: [Finishes #88077334]

File: processes/ripple_billing.js [hunk 1]
```diff
@@ -19,9 +19,90 @@
...
+    codius.logger.info("Ripple Billing Feature Enabled")
+
+    const RippleAccountMonitor = require('ripple-account-monitor')
+    const http                 = require('superagent')
+    const Promise              = require('bluebird')
+    const RIPPLE_REST_URL      = 'https://api.ripple.com/'
+    const ADDRESS              = codius.config.get('RIPPLE_ADDRESS')
+    const billing              = new codius.BillingService()
+    const CPU_PER_DROP         = 0.0075
+    const CPU_PER_BITCOIN      = 100000000
```
Comment (wilsonianb) [inline processes/ripple_billing.js]: Should this use `compute_units_per_bitcoin` from config.js? 
If so, also add `compute_units_per_drop` to the config.

Comment (stevenzeiler) [inline processes/ripple_billing.js]: yes I will update that now

**Comment by wilsonianb** [comment]
Does this need to be rebased?

File: lib/config.js [hunk 2]
```diff
@@ -37,6 +37,7 @@ nconf.defaults({
     "key": path.resolve(__dirname, '../server.key')
   },
   "compute_units_per_bitcoin": 45000000,
+  "compute_units_per_drop": 0.0075,
```
Comment (wilsonianb) [inline lib/config.js]: At .00005555 btc/xrp, I get `compute_units_per_drop` to be `0.0025`

**Comment by wilsonianb** [comment]
LGTM

**Comment by stevenzeiler** [comment]
+r

**Pull request merged**